apps/box/src/components/Cards/UsersCard.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  convertMegabyteToGigabyte,
+  FxBox,
+  FxButton,
+  FxCard,
+  FxSpacer,
+  FxTag,
+  FxText,
+} from '@functionland/component-library';
+import { CardRow, CardRowData, CardRowTitle } from './fields/CardRow';
+import { Image, StyleSheet } from 'react-native';
+import { CopyIcon } from '../Icons';
+import Clipboard from '@react-native-clipboard/clipboard';
+import { CardCarousel } from './fields/CardCarousel';
+import { Friend } from '../../api/users';
+import { getUserUsageDetails } from '../../utils/users';
+
+const USER_CARD_HEIGHT = 286;
+type UserCardType = React.ComponentProps<typeof FxCard> & {
+  data: Friend;
+};
+const UserCard = ({ data, ...rest }: UserCardType) => {
+  const usageStats = getUserUsageDetails(data);
+
+  const copyHandler = () => {
+    Clipboard.setString(data.decentralizedId);
+  };
+
+  return (
+    <FxCard {...rest}>
+      <FxBox flexDirection="row" alignItems="center">
+        <Image source={Number(data.imageUrl)} style={styles.image} />
```

Comment:
Why is source url wrapped with `Number()` function. I know `require()` returns a number which releates to an image from your local resources but if this is a URL it'll resolve as NaN.

Reply:
Going to leave this as is for the time being until we have integration with API to get image URL's